src/style.css

```diff
@@ -0,0 +1,90 @@
+@import url("./variables.css");
+@import url("./reset.css");
+@import url("./blocks/stars.css");
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__container {
+  display: flex;
+  flex-direction: column;
+  font-family: "Roboto", Arial, sans-serif;
+  color: var(--description-color-text);
+}
+
+.card__image {
+  margin-bottom: 40px;
+  align-self: center;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--description-color);
+}
+
+/* #region stars */
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__about-stars {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+/* #endregion */
+
+.card__price-value {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--description-color);
+}
+
+.card__value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  text-align: center;
+  padding: 12px 70px 12px 69px;
+  background: var(--button-colors);
+  border-radius: 5px;
+  text-transform: uppercase;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  transition: all 0.4s ease-out;
+}
+
+.card__button:hover {
+  background: #fff;
+  border: 1px solid var(--button-colors);
```

Comment:
don't add border in hover, add border to card__button with transparent color and change border-color in hover